### PR TITLE
feat: add list of regions and zones

### PR DESCRIPTION
**Pull request by Sh4d1**
Signed-off-by: Patrik Cyvoct <patrik@ptrk.io>

### CODE REVIEW

File: scw/region.go [hunk 1]
```diff
@@ -15,3 +15,21 @@ const (
 	ZoneFrPar1 = Zone("fr-par-1")
```
Comment (QuentinBrosse) [inline scw/region.go]: Please add the `fr-par-2` zone.

Comment (Sh4d1) [reply scw/region.go]: done

**Comment by QuentinBrosse** [comment]
I just moved `scw/region.go` to [`utils/locality.go`](https://github.com/scaleway/scaleway-sdk-go/blob/master/utils/locality.go) in #18 to avoid a circular dependancy.

Can you rebase your PR on `master` and refactor it, please?
I can handle it if you prefer. ;)

Thanks!

**Comment by Sh4d1** [comment]
Done :)

**Comment by QuentinBrosse** [comment]
We prefer:
- `AllRegions` instead of `Regions` 
- Replace `ZonesInRegion` constant by this getter: `func (r *Region) GetZones() []*Zone`
Ca irait pour ton use case ?

**Comment by Sh4d1** [comment]
:+1:  juste changed pointers to normal string

**Pull request merged**